logdna/logdna.py

```diff
@@ -100,27 +106,26 @@ def flush(self):
                         'tags': self.tags if self.tags else None},
                     stream=True,
                     timeout=self.request_timeout)
-                self.buf = []
-                self.buf_byte_length = 0
+                res.raise_for_status()
+
+                # when no RequestException happened
+                del self.buf[:]
```

Comment:
Let's use `self.buf.clear()` - more common

Reply:
Looks like clear works with python 3 only 
``` File "/usr/local/lib/python2.7/site-packages/logdna/logdna.py", line 87, in cleaning_after_success
    self.buf.clear()
AttributeError: 'list' object has no attribute 'clear'```

https://stackoverflow.com/questions/32055768/python3-attributeerror-list-object-has-no-attribute-clear

Reply:
`clear` works for `set`, according to [this](https://www.geeksforgeeks.org/set-clear-python/)